python/katana/warnings_config.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+
+def disable_partial_modin_warnings():
+
+    # Ignore any warnings related to defaulting to pandas for any operation.
+    warnings.filterwarnings("ignore", message=r".*[dD]efaulting to pandas implementation.*", module="modin")
+
+    # Ignore warnings related to distribution of object
```

Comment:
My understanding that filtering "ignores", but what I would like to see documented is the following:
* why does the warning happen in the first place
* why it is safe that we ignore it here

Reply:
That's fair, I will keep this in mind for the next one's we ignore as we hear from people.